NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -112,20 +117,36 @@ public void OpenJoinServerMenu(string serverIp)
 
             joinServerGameObject = new GameObject();
             JoinServer joinServerComponent = joinServerGameObject.AddComponent<JoinServer>();
-            string seperator = ":";
-            if (serverIp.Contains(seperator))
+
+            if (Regex.IsMatch(serverIp, "^[0-9.:]+$"))
```

Comment:
Can we break these into methods?  Like resolveIpv4 and resolveHostName.

Reply:
ipv6 is not covered with the regex

Reply:
The Lidgren methods we use doesn't support IPv6 too.